src/forms/inputs/checkbox-group.js

```diff
@@ -34,6 +36,7 @@ import {
  * @param {Object} meta - A `redux-form` [meta](http://redux-form.com/6.5.0/docs/api/Field.md/#meta-props) object
  * @param {Array} options - An array of checkbox values (strings, numbers, or key-value pairs)
  * @param {Object} [checkboxInputProps={}] - An object of key-value pairs representing props to pass down to all checkbox inputs
+ * @param {Boolean} [dropdown=false] - A boolean that determines whether the checkbox options are displayed in a dropdown container or not
```

Comment:
🍔 "{Boolean} - A boolean..." is redundant. That being said, we're pretty inconsistent in this repo about that!

Here's how it looks in the documentation:
![Image 2023-11-02 at 9 16 31 AM](https://github.com/LaunchPadLab/lp-components/assets/20426095/283a081b-0c04-44c2-a22c-d7a1a20f8266)